### DISCUSSION TITLE
Difference between Development client, Development build, Expo go

### DISCUSSION THREAD

**Discussion by cmd8** [expo/expo#23358] [Archived: Questions] [answered]
I'm confused with the terminology, could someone please clarify the below to me?
1. What is the difference between Development build and Expo Go? Also what is development client?
2. And how all above relate to different commands, e.g.:
- `expo start ` vs `expo start --dev-client`
- `expo start` vs `expo run:android` or `expo run:ios`
- `expo prebuild` vs `eas build`

:confused:

**Comment by eybel** [comment]
Same here, cant find the asnwer. Not sure if I should Switch "s" (in terminal) or not

**Comment by brentvatne** [comment, marked as answer]
hi there! check out this docs page: https://docs.expo.dev/workflow/overview/ - let me know if anything is unclear

you can think of dev client and dev build as synonymous. expo-dev-client is the library that you include in a development build that provides various dev tools such as the dev menu, app launcher, and eas update integration

**Comment by usernamedd** [comment]
is dev client like expo go ? can i install dev client on my physical device then i will install development build from a QR code that is provided by "exp run:android" command?

**Reply by sumitsharma7667** [reply]
Yes, the Expo Development Client is similar to Expo Go but supports custom native code. Install it on your device, create a development build using eas build -p android --profile development, and install it via the QR code provided by exp run:android.
